layouts/partials/footer.html

```diff
@@ -1,10 +1,11 @@
-{{ with site.Copyright }}
 {{ $year := now.Year }}
 <footer class="footer">
   <div class="container">
     <p>
-      &copy; {{ $year }} {{ . }}
+      {{ with site.Copyright }}
+      &copy; {{ $year }} {{ . }} </br>
+      {{ end }}
```

Comment:
This renders oddly on the page. I don't know if the CNCF holds trademarks independently from the LF, and I would avoid adding custom boilerplate without clear guidance from legal.

Reply:
I think the odd rendering is an artifact of the page responsiveness--this element tracks with the header, and not the content, so for some viewports it can look odd.

I think we should probably open an issue for responsiveness generally, and probably one for mobile rendering specifically.

Reply:
> I don't know if the CNCF holds trademarks independently from the LF, and I would avoid adding custom boilerplate without clear guidance from legal.

The double copyright lines has confused me for a couple weeks now. I see it on a lot of our project pages. I can try to find the issue i mentioned it in before.

Specifically for the template though, the "Cloud Native Computing Foundation" comes in from the config.toml, which we are expecting people to change when they generate a site from the template, so in practice we'd never actually see both of:
> © 2020 Cloud Native Computing Foundation
> © 2020 The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our Trademark Usage page.

So because the CNCF copyright mark _should_ be removed as a part of the initial site setup work, I think I'm OK leaving this as is.

That all said, how do we get that clear guidance from legal?

Reply:
@zacharysarah – What renders as `© 2020 Cloud Native Computing Foundation` is actually `© 2020 ProjectName` in reality. So for Longhorn, this would be `© 2020 Longhorn`, for Vitess `© 2020 Vitess`, etc.

What I suggest for this, @nate-double-u:

1) Change the string to render to `© 2020 The ProjectName Authors | Documentation Distributed under CC-BY-4.0`, which is what we actually use on 98% of projects.

The reason the second copyright line is there: the first copyright line (`ProjectName Authors`) == covers the actual code and docs. The second copyright line (`Linux Foundation`) covers the logos and trademarks.

Reply:
That makes sense. I'll add that change in -- however, @zacharysarah do we still need legal review, and if so, at which stage of the process do we need it? 

It's already published at https://cncf-hugo-starter.netlify.app/, as follows:
![Screen Shot 2020-10-29 at 2 40 58 PM](https://user-images.githubusercontent.com/4453979/97635599-827dce00-1a2f-11eb-9304-9568bc5d04a6.png)

Reply:
>     1. Change the string to render to `© 2020 The ProjectName Authors | Documentation Distributed under CC-BY-4.0`

@celestehorgan is it reasonable/safe, at least in the template here, to use the `site.Title` parameter within that string in place of "ProjectName"? (see my [edit here](https://github.com/cncf/hugo-netlify-starter/pull/40/files#diff-a13d08fb2a6877b0192f75489186a26d527b04bad2b4378ee4daf2be44848201))

Reply:
If we do it that way, likely we can take the `copyright` line out of the `config.toml` file.

Reply:
@nate-double-u I would actually do the following:

1. Rename `copyright` to `projectName` [here](https://github.com/cncf/hugo-netlify-starter/blob/main/config.toml#L3) (to make it clearer)

2. Reference `projectName` in the copyright footer, but also anywhere else we might use it.

Let `title` be its own thing that populates a page's `<title>` tag, i.e.

The reason I'd do it this way is because sometimes sites SEO-ify their `<title>` tag, and so we can't be guaranteed it's going to be a clean string of the project name every time.

Reply:
@celestehorgan that sounds good, I'll make that update.